sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/MetricsData.cs

```diff
@@ -41,6 +41,8 @@ metricType switch
                 MetricType.LongGauge => true,
                 MetricType.LongSum => true,
                 MetricType.Histogram => true,
+                MetricType.DoubleSumNonMonotonic => true,
```

Comment:
the whole IsSupportedType seems not relevant? Might be good to clean up later. Not blocker.

Reply:
I had the same question. It's being used to skip some extra processing.

https://github.com/Azure/azure-sdk-for-net/blob/71ce2a14eff9871622d20914415a297b188e3e72/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/MetricHelper.cs#L22-L39

Reply:
This was added initially to skip long types. But since, we added that later this is no longer needed